lib/reporters/base.js

```diff
@@ -34,9 +34,9 @@ exports = module.exports = Base;
  * Enable coloring by default, except in the browser interface.
  */
 
-exports.useColors = process.env
-  ? (supportsColor || (process.env.MOCHA_COLORS !== undefined))
-  : false;
+exports.useColors = process.browser
+  ? false
+  : (supportsColor || (process.env.MOCHA_COLORS !== undefined));
```

Comment:
Ah yeah, I just kept it that way because `process.browser` is `undefined` in Node, but I'll stop being lazy